src/types/invites/invite_stage_instance.ts

```diff
@@ -0,0 +1,12 @@
+import { GuildMember } from "../members/guild_member.ts";
+
+export interface InviteStageInstance {
+  /** The members speaking in the Stage */
+  members: Partial<GuildMember>;
```

Comment:
This doesnt look right can u double check this plz

Reply:
1. members but not an array?
2. what is it giving? Please double check because `{}` this is valid as per ur typings.
3. Any snowflakes that need to be made into bigints? because invites get cached